pkg/kubelet/apis/config/types.go

```diff
@@ -353,6 +353,10 @@ type KubeletConfiguration struct {
 	// The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics,
 	// rather than being surprised when they are permanently removed in the release after that.
 	ShowHiddenMetricsForVersion string
+	// LogFormat Flag specifies the structure of log messages.
+	// this flag accept `text` and `json` options
```

Comment:
We still haven't implemented `json` option. Let's describe current state. 
It would be good to describe that `text` is the default

Reply:
```
	// LogFormat specifies the structure of log messages.
	// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
	// Default: "text"
	// + optional
	LogFormat string `json:"logFormat,omitempty"`
```
changed to this